### DISCUSSION TITLE
Latest Tic80 is slow.

### DISCUSSION THREAD

**Discussion by aykaramba** [nesbox/TIC-80#2230] [General]
Hi guys.

I just wanted to drop and ask if the latest Tic80 is slow for anyone else?  I am on Debian 12 and I just git cloned and compiled Tic80.  When I surf or open up any game, the game runs at about 1/2 to 1/3 of the speed of my trusty 0.90 version.

Anyone else run into this issue?  Anyone know if there is a fix for this?

**Comment by nesbox** [comment]
Try to check the latest dev version https://nightly.link/nesbox/TIC-80/workflows/build/main

**Reply by aykaramba** [reply]
That works great!  Thank you very kindly.
